pkg/istio-agent/sds-agent.go

```diff
@@ -120,6 +119,8 @@ const (
 	InitialBackoff = "INITIAL_BACKOFF_MSEC"
 
 	pkcs8Key = "PKCS8_KEY"
+
+	CACertNamespaceConfigMapDataName = "ca-cert-ns.pem"
```

Comment:
Can we move this to `/pkg/config/constants`?

Reply:
done